### PR TITLE
EG prepare for release

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Review by xiangyan99 — APPROVED** [review]
Given this is a GA release, please make sure you get approval on API view.

File: sdk/eventgrid/azure-eventgrid/README.md [hunk 1]
```diff
@@ -102,6 +102,12 @@ Please have a look at the [samples](https://github.com/Azure/azure-sdk-for-pytho
 
  **Note:** It is important to know if your topic supports CloudEvents or EventGridEvents before publishing. If you send to a topic that does not support the schema of the event you are sending, send() will throw an exception.
 
+### System Topics
+A **[system topic](https://docs.microsoft.com/azure/event-grid/system-topics)** in Event Grid represents one or more events published by Azure services such as Azure Storage and Azure Event Hubs. For example, a system topic may represent all blob events or only blob created and blob deleted events published for a specific storage account.
+
+The names of the various event types for the system events published to Azure Event Grid are avaialable in `azure.eventgrid.SystemEventNames`.
```
Comment (swathipil) [inline sdk/eventgrid/azure-eventgrid/README.md:108]: typo:
```suggestion
The names of the various event types for the system events published to Azure Event Grid are available in `azure.eventgrid.SystemEventNames`.
```

File: sdk/eventgrid/azure-eventgrid/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 2.0.0b6 (Unreleased)
+## 2.0.0 (2021-03-09)
 
   **Breaking Changes**
+  - This version will be the last version to officially support Python 3.5, future versions will require Python 2.7 or Python 3.6+
```
Comment (yunhaoling) [inline sdk/eventgrid/azure-eventgrid/CHANGELOG.md:6]: I don't think this one belongs to be breaking changes sections :)

File: sdk/eventgrid/azure-eventgrid/README.md [hunk 3]
```diff
@@ -102,6 +102,12 @@ Please have a look at the [samples](https://github.com/Azure/azure-sdk-for-pytho
 
  **Note:** It is important to know if your topic supports CloudEvents or EventGridEvents before publishing. If you send to a topic that does not support the schema of the event you are sending, send() will throw an exception.
 
+### System Topics
+A **[system topic](https://docs.microsoft.com/azure/event-grid/system-topics)** in Event Grid represents one or more events published by Azure services such as Azure Storage and Azure Event Hubs. For example, a system topic may represent all blob events or only blob created and blob deleted events published for a specific storage account.
```
Comment (yunhaoling) [inline sdk/eventgrid/azure-eventgrid/README.md:106]: nit: use noun for create and delete?
```suggestion
A **[system topic](https://docs.microsoft.com/azure/event-grid/system-topics)** in Event Grid represents one or more events published by Azure services such as Azure Storage or Azure Event Hubs. For example, a system topic may represent all blob events or only blob creation and blob deletion events published for a specific storage account.
```

**Pull request merged**